open-machine-learning-jupyter-book/assignments/data-science/Soda-profits.ipynb

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
<p>put the install always as the first code cell. Also split the % code to a saparate cell.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>